intranet/static/js/dashboard/seniors.js

```diff
@@ -1,49 +1,49 @@
 /* global $ */
-function getTimeRemaining(endtime){
-  var t = Date.parse(endtime) - Date.parse(new Date());
-  var seconds = Math.floor( (t/1000) % 60 );
-  var minutes = Math.floor( (t/1000/60) % 60 );
-  var hours = Math.floor( (t/(1000*60*60)) % 24 );
-  var days = Math.floor( t/(1000*60*60*24) );
-  return {
-    'total': t,
-    'days': days,
-    'hours': hours,
-    'minutes': minutes,
-    'seconds': seconds
-  };
+function getTimeRemaining(endtime) {
+    var t = Date.parse(endtime) - Date.parse(new Date());
+    var seconds = Math.floor((t / 1000) % 60),
+        minutes = Math.floor((t / 1000 / 60) % 60),
+        hours = Math.floor((t / (1000 * 60 * 60)) % 24),
+        days = Math.floor(t / (1000 * 60 * 60 * 24));
+    return {
+        'total': t,
+        'days': days,
+        'hours': hours,
+        'minutes': minutes,
+        'seconds': seconds
+    };
 }
 
-function initClock(id, evt, endtime){
-  var clock = document.querySelector(id);
-  if(!clock) return;
+function initClock(id, evt, endtime) {
+    var clock = document.querySelector(id);
+    if (!clock) return;
 
-  function updateClock() {
-    var t = getTimeRemaining(endtime);
+    function updateClock() {
+        var t = getTimeRemaining(endtime);
 
-    var dys = t.days;
-    var hrs = t.hours;
-    var mns = t.minutes;
-    var scs = ('0' + t.seconds).slice(-2);
+        var dys = t.days,
+            hrs = t.hours,
+            mns = t.minutes,
+            scs = ('0' + t.seconds).slice(-2);
 
-    if(dys > 50) {
-        clock.innerHTML = "<span class='clock'><b>"+dys+"</b> days, <b>"+hrs+"</b> hours</span> until " + evt;
-    } else {
-        clock.innerHTML = "<span class='clock'><b>"+dys+"</b> days, <b>"+hrs+"</b> hours, <b>"+mns+"</b> minutes, <b>"+scs+"</b> seconds</span><br/>until " + evt;
-    }
+        if (dys > 50) {
+            clock.innerHTML = "<span class='clock'><b>" + dys + "</b> days, <b>" + hrs + "</b> hours</span> until " + evt;
+        } else {
+            clock.innerHTML = "<span class='clock'><b>" + dys + "</b> days, <b>" + hrs + "</b> hours, <b>" + mns + "</b> minutes, <b>" + scs + "</b> seconds</span><br/>until " + evt;
+        }
 
-    if(t.total<=0){
-      clearInterval(timeinterval);
-      clock.innerHTML = "<span class='clock'><b>YOU HAVE GRADUATED!</b></span>";
+        if (t.total <= 0) {
+            clearInterval(timeinterval);
+            clock.innerHTML = "<span class='clock'><b>YOU HAVE GRADUATED!</b></span>";
+        }
     }
-  }
 
-  updateClock();
-  var timeinterval = setInterval(updateClock, 500);
+    updateClock();
+    var timeinterval = setInterval(updateClock, 500);
 }
 
 $(function() {
     var graddate = $(".seniors-widget").attr("data-graduation-date");
     var deadline = graddate + " GMT-0400 (EDT)";
     initClock(".seniors-clock", "Graduation!", deadline);
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2747127034/issues/source?bid=3176160&fileBranchId=3355598#l49)